data/hai/hai missions.txt

```diff
@@ -209,6 +209,79 @@ mission "Ask the Hai about the Unfettered"
 
 
 
+mission "Hai Sympathizer Warning"
+	invisible
+	source
+		government "Hai"
+		not attributes "uninhabited" "retirement"
+	to offer
+		"reputation: Hai Merchant (Sympathizers)" < 0
+		not "warned about hai merchant politics"
```

Comment:
Are these conditions used anywhere else?

Reply:
I think they're good to have for later

Reply:
Not yet, but in theory there are a few different ways you could potentially end up receiving warnings which I'd like to expand on over time.
I'd also like to be able to increment it for subsequent warnings after some stuff.
It's a condition I'd *like* to use, and like to not have to make a patch mission for later on.